app/views/docs/realtime.phtml

```diff
@@ -405,10 +405,10 @@ subscription.close()</code></pre>
                 timestamp
             </td>
             <td data-title="Type: ">
-                integer
+                string
             </td>
             <td data-title="Description: ">
-                The <a href="https://en.wikipedia.org/wiki/Unix_time" rel="noopener" target="_blank">UNIX timestamp</a> from the server to ensure consistency across all client platforms and real-time technologies.
+                The <a href="https://en.wikipedia.org/wiki/ISO_8601" rel="noopener" target="_blank">ISO 8601</a> from the server to ensure consistency across all client platforms and real-time technologies.
```

Comment:
Please modify the description to let readers know that the timestamp is in UTC timezone for consistency :)

Reply:
Sure